pkg/api/meta/restmapper.go

```diff
@@ -335,3 +359,23 @@ func (m MultiRESTMapper) AliasesForResource(alias string) (aliases []string, ok
 	}
 	return nil, false
 }
+
+// ResourceInGroup verifies if the group has the resource in it
+func (m MultiRESTMapper) ResourceInGroup(group, resource string) bool {
+	for _, t := range m {
+		if t.ResourceInGroup(group, resource) {
+			return true
+		}
+	}
+	return false
+}
+
+// IsResource takes a string and checks if it's a resource
```

Comment:
@deads2k this is what we were discussing before.

Prioritization of groups is implicit in the order of the MultiRESTMapper.  We may be lacking a way to indicate to a caller of the MultiRESTMapper.IsResource that the match was ambiguous, as well as a method on MultiRESTMapper that accepts resource + group.